assets/constants.ts

```diff
@@ -5054,132 +5054,132 @@ export default {
     "direction": "ltr"
   }, {
     "id": "ak",
-    "description": "Akan",
+    "description": "Ákán (Akan)",
```

Comment:
Should we add a test that will make sure that language with the same code has the same description?

Reply:
I think the description of the language can be different according to the place where it is being used, maybe if it used to be displayed in the language dropdown to show supported site language, it can be different from the one shown in the voiceover dropdown.

Reply:
Okay